mllib-dal/src/main/java/com/intel/oap/LibLoader.java

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package com.intel.oap.mllib;
+package com.intel.oap;
```

Comment:
Let's keep this inside oap mllib namespace. oap space is for the whole oap product. same below.